Dockerfile

```diff
@@ -1,6 +1,24 @@
-ARG ROS2_VERSION=humble
-FROM ghcr.io/aica-technology/ros2-ws:${ROS2_VERSION} as base
-USER ${USER}
+ARG BASE_TAG=22.04
+FROM ubuntu:${BASE_TAG} as base
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update && apt-get install -y \
+    cmake \
+    g++ \
+    git \
+    libgtest-dev \
+    libeigen3-dev \
+    python3-pip \
+    ssh \
+    sudo \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN echo "Set disable_coredump false" >> /etc/sudo.conf
```

Comment:
Nit: I reckon this and `RUN echo "session required pam_limits ...` can also be moved to the `development` stage as they are specifically for user management